### PR TITLE
Add Team Forms certified connector

### PR DESCRIPTION
**Pull request by AnthonyPhan**
- [x] I attest that the connector doesn't exist on the Power Platform today. I've verified by checking the pull requests in GitHub and by searching for the connector on the platform or in the documentation. 
- [x] I attest that the connector works and I verified by deploying and testing all the operations. 
- [x] I attest that I have added detailed descriptions for all operations and parameters in the swagger file.
- [x] I attest that I have added response schemas to my actions, unless the response schema is dynamic. 
- [x] I validated the swagger file, `apiDefinition.swagger.json`, by running `paconn validate` command.
- [x] If this is a certified connector, I confirm that `apiProperties.json` has a valid brand color and doesn't use an invalid brand color, `#007ee5` or `#ffffff`. If this is an independent publisher connector, I confirm that I am not submitting a connector icon.

### CODE REVIEW

**Comment by AnthonyPhan** [comment]
@microsoft-github-policy-service agree [company="VP Labs"]

**Comment by AnthonyPhan** [comment]
@microsoft-github-policy-service agree company="VP Labs"

File: certified-connectors/Team Forms/README.md [hunk 1]
```diff
@@ -0,0 +1,63 @@
...
+
+Returns all the forms for a given team.
+
+### `Action` Form
+
+Returns details for a specific form.
+
+### `Action` Files
+
+Returns meta data for all files/attachments associated with a form response. Users can use the id from the response with the [SharePoint actions]('https://learn.microsoft.com/en-us/connectors/sharepointonline/') to get the file contents.
```
Comment (Amjed-Ayoub) [inline certified-connectors/Team Forms/README.md:35]: @AnthonyPhan all files should be free of grammatical errors and typos. can you please change "meta data" to "metadata". please fix in all your files.

File: certified-connectors/Team Forms/README.md [hunk 2]
```diff
@@ -0,0 +1,63 @@
...
+
+Returns details of a form response.
+
+### `Action` Response Schema
+
+This action is used internally to dynamically get a form response schema so flow can provide intuitive response fields. You cannot call this action directly.
+
+### `Action` Delete Subscription
+
+This action is used internally to delete subscription trigger when flow trigger actions are updated of deleted.
```
Comment (Amjed-Ayoub) [inline certified-connectors/Team Forms/README.md:55]: @AnthonyPhan all files should be free of grammatical errors and typos. can you please change "updated of deleted" to "updated or deleted".

File: certified-connectors/Team Forms/README.md [hunk 3]
```diff
@@ -0,0 +1,63 @@
...
+
+## Obtaining Credentials
+
+To use Team Forms simply logon using your existing Microsoft Office work or school account and provide it access to SharePoint and Teams.
```
Comment (Amjed-Ayoub) [inline certified-connectors/Team Forms/README.md:59]: @AnthonyPhan all files should be free of grammatical errors and typos. can you please change "logon" to "login".

File: certified-connectors/Team Forms/apiDefinition.swagger.json [hunk 4]
```diff
@@ -0,0 +1,674 @@
+{
+  "swagger": "2.0",
+  "info": {
+    "title": "Team Forms",
+    "description": "Team Forms brings digital forms into Microsoft Teams with no code.",
```
Comment (Amjed-Ayoub) [inline certified-connectors/Team Forms/apiDefinition.swagger.json:5]: @AnthonyPhan I would highly suggest you use the same description from your readme file.

File: certified-connectors/Team Forms/apiProperties.json [hunk 5]
```diff
@@ -0,0 +1,8 @@
+{
+  "properties": {
```
Comment (Amjed-Ayoub) [inline certified-connectors/Team Forms/apiProperties.json:2]: @AnthonyPhan please follow our template [here](https://github.com/microsoft/PowerPlatformConnectors/blob/dev/templates/certified-connectors/apiProperties.json) for your apiProperties.json. You can replace the secrets with dummy data.

**Review by Amjed-Ayoub — CHANGES_REQUESTED** [review]

File: certified-connectors/Team Forms/README.md (hunk 1 above)
Comment (AnthonyPhan) [reply certified-connectors/Team Forms/README.md]: Thank you, fixed.

File: certified-connectors/Team Forms/README.md (hunk 3 above)
Comment (AnthonyPhan) [reply certified-connectors/Team Forms/README.md]: Thank you, fixed.

File: certified-connectors/Team Forms/apiProperties.json (hunk 5 above)
Comment (AnthonyPhan) [reply certified-connectors/Team Forms/apiProperties.json:2]: Thank you, fixed.

**Review by Amjed-Ayoub — APPROVED** [review]

**Comment by Amjed-Ayoub** [comment]
Hello @AnthonyPhan,
I hope you are doing well.
Congratulations, your pull request is approved and merged. Please [submit the files to ISV Studio](https://docs.microsoft.com/en-us/connectors/custom-connectors/submit-for-certification#submit-to-isv-studio) or [update your submission](https://docs.microsoft.com/en-us/connectors/custom-connectors/certification-updates) using your latest commit ID, please allow us up to 1-2 weeks to review your submission, and our Engineers will notify you in the “Activity Control” area in ISV Studio. Please [make sure that the icon meets all the requirements](https://docs.microsoft.com/en-us/connectors/custom-connectors/certification-submission#design-an-icon-for-your-connector), that it’s size is 230x230, icon background is not white or transparent and it matches the iconBrandColor property in apiProperties.json file.
After making the submission in ISV Studio, please attach an intro.md file in the activity control section. This file is required and it's different from readme.md. You can [find an example of intro.md file on this page](https://docs.microsoft.com/en-us/connectors/custom-connectors/submit-for-certification#submit-to-isv-studio) at step 6.
Please [Create an environment in the Preview region](https://docs.microsoft.com/en-us/connectors/custom-connectors/certification-testing#create-an-environment-in-the-preview-region). You will use this environment later to [test your connector](https://docs.microsoft.com/en-us/connectors/custom-connectors/certification-testing#test-your-connector) after Microsoft is done with the functional verification for your connector. 
We expect all tests to be completed within 1-2 weeks.
If your connector passes all tests, the deployment process begins, and it typically takes up to 3 to 4 weeks to deploy your connector to all of our regions.
Please let me know if you have any questions or run into any problems while creating or updating your submission.
Thank you very much for working with us.

**Pull request merged**